src/modules/commander/HealthAndArmingChecks/checks/cpuResourceCheck.cpp

```diff
@@ -61,18 +67,20 @@ void CpuResourceChecks::checkAndReport(const Context &context, Report &reporter)
 
 	} else {
 		const float cpuload_percent = cpuload.load * 100.f;
+		const bool high_cpu_load = cpuload_percent > _param_com_cpu_max.get();
+		_high_cpu_load_hysteresis.set_state_and_update(high_cpu_load, hrt_absolute_time());
 
-		if (cpuload_percent > _param_com_cpu_max.get()) {
-
+		// fail check if CPU load is above the threshold for 2 seconds
+		if (_high_cpu_load_hysteresis.get_state()) {
 			/* EVENT
-			 * @description
-			 * The CPU load can be reduced for example by disabling unused modules (e.g. mavlink instances) or reducing the gyro update
-			 * rate via <param>IMU_GYRO_RATEMAX</param>.
-			 *
-			 * <profile name="dev">
-			 * The threshold can be adjusted via <param>COM_CPU_MAX</param> parameter.
-			 * </profile>
-			 */
+			* @description
```

Comment:
Indentation was already broken with a1e71097ba0ccaca8b73d17f9762ebd8b05c0c80 and I didn't even notice.

Reply:
Corrected by amending my commit.